packages/terra-form-input/src/Input.scss

```diff
@@ -0,0 +1,92 @@
+    line-height: var(--terra-form-input-line-height, 1.143);
+    opacity: var(--terra-form-input-opacity, 1);
+    padding-bottom: var(--terra-form-input-padding-bottom, 0.571rem);
+    padding-left: var(--terra-form-input-padding-left, 0.357rem);
+    padding-right: var(--terra-form-input-padding-right, 0.357rem);
+    padding-top: var(--terra-form-input-padding-top, 0.571rem);
+    transition-duration: var(--terra-form-input-transition-duration, 0s);
+    transition-property: background-size;
+    transition-timing-function: var(--terra-form-input-transition-timing-function, ease);
+    width: 100%;
+
+    &:hover {
+      border: var(--terra-form-input-hover-border, 1px solid #4a4a4a);
+    }
+
+    &[required] {
+      box-shadow: unset;
+    }
+
+    &[disabled] {
+      background-color: var(--terra-form-input-disabled-background-color, rgba(0, 0, 0, 0.05));
+      border: var(--terra-form-input-disabled-border, 1px solid #dedfe0);
+      color: var(--terra-form-input-disabled-color, rgba(0, 0, 0, 0.25));
+      cursor: default;
+      opacity: var(--terra-form-input-disabled-opacity, 1);
+    }
+
+    &:focus {
+      background-color: var(--terra-form-input-focus-background-color, #fff);
+      background-size: var(--terra-form-input-focus-keyboard-animation-size, 0 0);
+      border: var(--terra-form-input-focus-border, 1px solid #4a4a4a);
+      border-radius: var(--terra-form-input-focus-border-radius, 0.2rem);
+      opacity: var(--terra-form-input-focus-opacity, 1);
+      outline: none;
+
+      [dir=ltr] & {
+        box-shadow: var(--terra-form-input-focus-keyboard-box-shadow-ltr, 0 0 8px #a6d9f4);
+      }
+
+      [dir=rtl] & {
+        box-shadow: var(--terra-form-input-focus-keyboard-box-shadow-rtl, 0 0 8px #a6d9f4);
+      }
+    }
+  }
+
+  .form-error {
+    background-image: var(--terra-form-input-error-focus-animation-background, none);
+    border: var(--terra-form-input-error-border, 1px solid #ff0b0f);
+    border-radius: var(--terra-form-input-error-border-radius, 0.2rem);
+    opacity: var(--terra-form-input-error-opacity, 1);
+
+    &:hover {
+      border: var(--terra-form-input-error-hover-border, 1px solid #ff0b0f);
+    }
+
+    &[disabled] {
+      background-color: var(--terra-form-input-error-disabled-background-color, rgba(0, 0, 0, 0.05));
+      border: var(--terra-form-input-error-disabled-border, 1px solid #ff0b0f);
+      color: var(--terra-form-input-error-disabled-color, rgba(0, 0, 0, 0.25));
+      opacity: var(--terra-form-input-error-disabled-opacity, 1);
+    }
+
+    &:focus {
+      background-color: var(--terra-form-input-error-focus-background-color, #fff);
+      background-size: var(--terra-form-input-error-focus-keyboard-animation-size, 0 0);
+      border: var(--terra-form-input-error-focus-border, 1px solid #ff0b0f);
+      border-radius: var(--terra-form-input-error-focus-border-radius, 0.2rem);
+      opacity: var(--terra-form-input-error-focus-opacity, 1);
```

Comment:
Check with @neilpfeiffer on this, I'm thinking if there is an opacity change, it happens at background or color style with rgba instead of changing opacity.

Reply:
I can change this if Neil wants, I just kept this consistent with the `form-textarea` component.

Reply:
Changed 44e3f71e18905ce93d017024abfc1047c92e6b98